drivers/cache/Kconfig.aspeed

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2022 ASPEED Technology Inc.
+
+config CACHE_ASPEED
+bool "ASPEED external cache driver"
+	depends on SOC_SERIES_AST10X0
+	select CACHE_HAS_DRIVER
```

Comment:
Sorry, maybe I was not clear enough. You should add here `depends on SYSCON` since your driver is actually depending on some syscon API.

Reply:
Got it.  Force push the modified commit.